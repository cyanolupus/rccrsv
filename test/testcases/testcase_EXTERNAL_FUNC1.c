int foo(); int bar(int a, int b); int testcase() { return foo() + bar(3,5); }
