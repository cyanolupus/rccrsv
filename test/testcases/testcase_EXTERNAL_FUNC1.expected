Hello, World!
return 131
